debian/control

```diff
@@ -13,13 +13,15 @@ Depends: ${misc:Depends}, ${python3:Depends},
  python3 (>= 3.10.0),
  python3-xlib,
  python3-babel,
- x11-utils,
- xprintidle,
- alsa-utils,
  python3-croniter,
  python3-packaging,
  gir1.2-notify-0.7,
  gir1.2-gtk-4.0
+Recommends:
+ python3-pywayland,
+ ffmpeg | pipewire
```

Comment:
Since `alsa-utils` was in `Depends`, should we also use `ffmpeg | pipewire` there? Does that work?

Reply:
Yes, that should also work